### ISSUE TITLE
Fix UI issues as a result of now showing 0s for "zero data"

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#4226]
#4085 and #4086 modified the behavior for all Search Console- and Analytics-based widgets to only show the "gathering data" state when the service is actually gathering data, fixing a long-standing problem. If a site has data but they're all 0s, this is now displayed in the regular widgets.

While this is (hopefully) a rare case experience, we should still make sure the current UI for that is not "terrible". Currently, by "only" implementing #4085 and #4086 there are a few ugly UI problems though that we should resolve. While there is going to be a more significant revamp of different variants of "zero data" states at another point in the near future, this issue is merely about resolving a few problems in the _current_ "all 0s" UI which as of #4085 and #4086 will be shown on applicable sites.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

### Test Coverage

* 

## QA Brief

* 

## Changelog entry

*

**Comment by felixarntz** [comment]
@eugene-manuilov Fair point - IB ✅

**Comment by eugene-manuilov** [comment]
@felixarntz this is ready for code review. I have included more changes in my PR that fix some edge cases and broken storybook stories.

**Comment by felixarntz** [comment]
@eugene-manuilov Can you add a QA Brief please?

**Comment by cole10up** [comment]
Question with the above @felixarntz - we're showing the '^ 100%' green item on the widget graph date range above. Is this expected? Should we adjust this?

**Comment by felixarntz** [comment]
@cole10up Good catch! It's not critical to fix in this issue though, can you open a follow-up issue please to change this to 0%?
